packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -35,7 +36,20 @@ function getWorkerTests(opts: { experimental: boolean }): WorkerTestConfig[] {
 		return [
 			{
 				template: "hello-world",
-				variants: ["TypeScript", "JavaScript", "Python"],
+				variants: ["TypeScript", "JavaScript"],
+				verifyDeploy: {
+					route: "/",
+					expectedText: "Hello World!",
+				},
+				verifyPreview: {
+					route: "/",
+					expectedText: "Hello World!",
+				},
+				verifyTest: true,
```

Comment:
this feels like something we should be doing for all C3 starter projects that come accompanied by tests?

Reply:
Afaik only hello-world and the experimental hello-world-with-assets template (js/ts) includes some tests. But there is no e2e tests for non-framework experimental templates. Let me see if I can set it up for the assets template 👍🏼

Reply:
Added tests for all non-framework experimental templates on https://github.com/cloudflare/workers-sdk/pull/7827/commits/68656739cf26984f8d21d4e235694982d42259d1 and https://github.com/cloudflare/workers-sdk/pull/7827/commits/c180e936b4e737d2c565bb302de7fc1e28c29705

Reply:
@edmundhung you are the abs best!!!! ♥️